chrome-app/links.js

```diff
@@ -18,41 +18,64 @@ const P_DESCRIPTIONS = {
 
 const pr = ["is:open", "is:pr", "archived:false"];
 const issue = ["is:open", "is:issue", "archived:false"];
-const ourRepos = ["repo:aws/jsii", "repo:aws/aws-cdk", "repo:awslabs/aws-cdk-rfcs", "repo:awslabs/cdk-ops", "repo:aws-samples/aws-cdk-examples"];
+const ourRepos = ["repo:aws/jsii", "repo:aws/aws-cdk", "repo:aws/aws-cdk-rfcs", "repo:awslabs/cdk-ops", "repo:aws-samples/aws-cdk-examples"];
 const hideInProgress = [ "-label:status/in-progress" ];
 const sortByOldestFirst = [ "sort:created-asc" ];
 
 // LINKS
 const LINKS = [
+  [
+    { title: "Triage", classes: "narrow" },
+    {
+      title: "B",
+      href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", 
+        "label:bug",
+        "-label:p0", "-label:p1", "-label:p2", "-label:p3" , 
+        "-label:effort/small", "-label:effort/medium", "-label:effort/large",
+        "-label:guidance",
+      ]),
+      description: "For bugs, label with priority",
+    },
+    {
+      title: "F",
+      href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", 
+        "-label:bug",
```

Comment:
Why not +feature-request?

Reply:
I wanted to make sure we don't miss anything so I decided that anything that is not a bug or guidance will go under "feature requests". Just trying to keep it simple.

Reply:
I would say that's not keeping it simple. Keeping it simple would be assuming everything was labeled properly (and maybe adding a filter for unlabeled items).

We actually have some stuff in there that legitimately doesn't fall under any of the banners (and obviously some actual items that never got classified): https://github.com/aws/aws-cdk/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3Abug+-label%3Afeature-request+-label%3Aguidance

But fine I guess.